src/usexp/usexp.ml

```diff
@@ -100,21 +127,26 @@ let split_string s ~on =
   in
   loop 0 0
 
+let pp_print_atom ppf ~serialize s =
+  if String.contains s '\n' then begin
+    match split_string s ~on:'\n' with
+    | [] -> Format.pp_print_string ppf (serialize s)
+    | first :: rest ->
+       Format.fprintf ppf "@[<hv 1>\"@{<atom>%s" (serialize first);
+       List.iter rest ~f:(fun s ->
+           Format.fprintf ppf "@,\\n%s" (serialize s));
+       Format.fprintf ppf "@}\"@]"
+  end else
+    Format.fprintf ppf "%S" (serialize s)
```

Comment:
Should be `"%s" (quote s)`. The escaping conventions for S-expressions are not exactly the same as OCaml ones.

Reply:
So for the line 137, there should be no quote (split string) but the behavior should be different for line 140 — the string does not contain newlines, so will not be split.

Reply:
Seems right